docs/source/usage/vex.rst

```diff
@@ -0,0 +1,94 @@
+============
+vex
+============
+
+.. argparse::
+    :filename: ./cdxev/__main__.py
+    :func: create_parser
+    :prog: cdx-ev
+    :path: vex
+
+    This command provides different operations on VEX-/SBOM- Files with embedded vulnerabilities. The vex-command has the following subcommands:
```

Comment:
VEX-/SBOM-file (if you want to keep it consistent with the text later one, where you also only refer to is as "SBOM-file". Though the "-" is not even required.